lib/couchdb_config.py

```diff
@@ -0,0 +1,169 @@
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+
+DOCUMENTATION = '''
+---
+module: couchdb_config
+short_description: Add and change CouchDB configuration keys
+description:
+    - This module allows you to change the CouchDB configuration. Currently supported is adding and changing a section key.
+    - This is equivalent to a PUT /_config/{section}/{key} using the CouchDB API (http://docs.couchdb.org/en/latest/api/server/configuration.html#config-section-key)
+version_added: 1.0
+author: Leif Hanack @leifhanack
+requirements: [ "pycouchdb", "requests" ]
+options:
+    section:
+        description:
+            - The configuration section name
+        required: true
+    key:
+        description:
+            - The configuration key name
+        required: true
+    value:
+        description:
+            - The configuration value
+        required: true
+    host:
+        description:
+            - The host running the database
+        required: false
+        default: localhost
+    port:
+        description:
+            - The port to connect to
+        required: false
+        default: 5984
+    user:
+        description:
+            - The administrator user used to authenticate with
+        required: false
+    password:
+        description:
+            - The administrator password of the user
+        required: false
+'''
+
+EXAMPLES = '''
+# setting multiple values
+- couchdb_config: >
+    section={{ item.section }}
+    key={{ item.key }}
+    value={{ item.value }}
+  with_items:
+    - { section: 'couch_httpd_auth', key: 'timeout', value: '60' }
+    - { section: 'socket_keys', key: 'worker_batch_size', value: '510' }
+
+# setting a list of tuples
+- couchdb_config: >
+    section='httpd'
+    key='socket_keys'
+    value='[{recbuf, 262140}, {sndbuf, 262140}]'
+
+# setting a value when CouchDB is secured
+- couchdb_config: >
+    user=admina
+    password=@dm1na
+    section='couch_httpd_auth'
+    key='timeout'
+    value='60'
+
+# changing host and port
+- couchdb_config: >
+    host=192.168.33.100
+    port=5985
+    section='couch_httpd_auth'
+    key='timeout'
+    value='60'
+'''
+
+IS_INSTALLED_PYCOUCHDB = True
+try:
+    import pycouchdb
+    from pycouchdb.resource import Resource
+except ImportError:
+    IS_INSTALLED_PYCOUCHDB = False
+
+IS_INSTALLED_REQUESTS = True
+try:
+    import requests
+    from requests.exceptions import ConnectionError
+except ImportError:
+    IS_INSTALLED_REQUESTS = False
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            host=dict(default="localhost"),
+            port=dict(type='int', default=5984),
+            user=dict(type='str'),
+            password=dict(type='str', no_log=True),
+            section=dict(type='str', required=True),
+            key=dict(type='str', required=True),
```

Comment:
It depends on where you look:) 

I based my wording on the REST API doc. See http://docs.couchdb.org/en/latest/api/server/configuration.html#config-section-key